eng/scripts/Language-Settings.ps1

```diff
@@ -62,34 +62,43 @@ function Get-java-PackageInfoFromRepo ($pkgPath, $serviceDirectory)
 # Returns the maven (really sonatype) publish status of a package id and version.
 function IsMavenPackageVersionPublished($pkgId, $pkgVersion, $groupId)
 {
-  try
+  $uri = "https://oss.sonatype.org/content/repositories/releases/$($groupId.Replace('.', '/'))/$pkgId/$pkgVersion/$pkgId-$pkgVersion.pom"
```

Comment:
I guess we had the but with the groupId before which you are fixing as well.

Reply:
Yeah.  The original url was using groupId as-is, but that produces broken urls.  Sontatype may have been rewriting the urls or handling them differently before, but now, groupIds have to be altered in urls.